docroot/themes/humsci/humsci_basic/src/js/main-nav/menu-toggle.js

```diff
@@ -0,0 +1,56 @@
+const menuToggle = document.querySelector('.su-main-nav__toggle');
+const mainMenu = document.querySelector('.su-main-nav__menu-lv1');
+const mobileNavBreakpoint = 992;
+let windowWidth = window.innerWidth;
+let isHidden;
+let wasDesktopSize;
+
+const collapseNav = () => {
+  menuToggle.setAttribute('aria-expanded', false);
+  mainMenu.setAttribute('aria-hidden', true);
+  menuToggle.innerHTML = "Menu";
+  isHidden = true;
+}
+
+const showNav = () => {
+  menuToggle.setAttribute('aria-expanded', true);
+  mainMenu.setAttribute('aria-hidden', false);
+  menuToggle.innerHTML = "Close";
+  isHidden = false;
+}
+
+if (menuToggle) {
+  // If below a certain breakpoint, hide the nav on page load
+  if (window.innerWidth < mobileNavBreakpoint) {
```

Comment:
I think this should already be set up by default in the template code and styles. I don't think we should use JS to accomplish it. I would use no-JS and .js styles to do so instead in the styles.